test/test_auth_adapter.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class TestAuthAdapter < Test::Unit::TestCase
+  def test_undefined_auth_adapter
+    flexmock(TCPSocket).should_receive(:new).ordered.with('ldap.example.com', 379).once.and_return(nil)
```

Comment:
Thank you for your feedback. I had no idea so I referred to `test/test_ldap_connection.rb`
What would you do if you were me in this case?